src/wombats/constants.clj

```diff
@@ -39,18 +39,52 @@
    102000 "Wombat with the name '%s' already exists."
    102001 "Wombat source code with that pathname has already registered. If you own the source code, change the file name and try again."})
 
-(defonce initial-stats
+(def initial-stats
   {:stats/frame-number 0
+   :stats/score 0
    :stats/food-collected 0
    :stats/poison-collected 0
-   :stats/score 0
+   :stats/number-of-moves 0
+   :stats/smoke-bombs-thrown 0
+   :stats/shots-fired 0
+   :stats/shots-hit 0
+   :stats/have-been-shot 0
    :stats/wombats-destroyed 0
-   :stats/wombats-hit 0
    :stats/zakano-destroyed 0
-   :stats/zakano-hit 0
+   :stats/wombats-shot 0
+   :stats/zakano-shot 0
    :stats/wood-barriers-destroyed 0
-   :stats/wood-barriers-hit 0
-   :stats/shots-fired 0
-   :stats/shots-hit 0
-   :stats/smoke-bombs-thrown 0
-   :stats/number-of-moves 0})
+   :stats/steel-barriers-destroyed 0
+   :stats/wood-barriers-shot 0
+   :stats/steel-barriers-shot 0
+   :stats/deaths 0
+   :stats/deaths-by-shot 0
+   :stats/deaths-by-wood-barrier-collision 0
+   :stats/deaths-by-steel-barrier-collision 0
+   :stats/deaths-by-wombat-collision 0
+   :stats/deaths-by-zakano-collision 0
+   :stats/deaths-by-poison 0
+   :stats/wood-barrier-collisions 0
+   :stats/steel-barrier-collisions 0
+   :stats/zakano-collisions 0
+   :stats/wombat-collisions 0
+   :stats/frames-blinded 0
+   :stats/frames-played 0})
```

Comment:
What's the difference between: `:stats/frame-number` and `:stats/frames-played`?